cmd/vaas-hook/main.go

```diff
@@ -45,6 +48,24 @@ func init() {
 	log.Printf("Initializing %s %s", AppName, Version)
 
 	Config = CommonConfig{}
+	podInfo, err := k8s.GetPodInfo()
```

Comment:
Please move this part (fetching pod info), to part of the code that is executed at least after calling --help.